cocos/ui/UIEditBox/UIEditBoxImpl-common.cpp

```diff
@@ -95,8 +99,39 @@ void EditBoxImplCommon::initInactiveLabels(const Size& size)
 
 void EditBoxImplCommon::placeInactiveLabels()
 {
-    _label->setPosition(CC_EDIT_BOX_PADDING, _contentSize.height / 2.0f);
-    _labelPlaceHolder->setPosition(CC_EDIT_BOX_PADDING, _contentSize.height / 2.0f);
+    switch (_textHorizontalAlignMent) {
+        case cocos2d::TextHAlignment::LEFT :
```

Comment:
@zilongshanren  In previous commit you check I called 
_label->setAlignment(_textHorizontalAlignment, _textVerticalAlignment);
    _labelPlaceHolder->setAlignment(_textHorizontalAlignment, _textVerticalAlignment);

method from placeInactiveLabels.